sql/functions/run_maintenance.sql

```diff
@@ -219,24 +219,24 @@ LOOP
         FOR v_row_max_time IN
             SELECT partition_schemaname, partition_tablename FROM @extschema@.show_partitions(v_row.parent_table, 'DESC', false)
         LOOP
-            EXECUTE format('SELECT max(%s)::text FROM %I.%I'
+            EXECUTE format('SELECT %s::text FROM %I.%I LIMIT 1'
```

Comment:
Update the comments on lines 217 above and 241 below to be relevant that it's no longer getting the max value. Other than that, looks great! Thanks for the PR!

Reply:
Looks. great. Thanks!